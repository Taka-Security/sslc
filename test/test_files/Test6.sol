contract A {
  struct StructA {
    bool var1;
  }
  struct StructB {
    uint8 var1;
    StructA var2;
    address payable var3;
  }
}