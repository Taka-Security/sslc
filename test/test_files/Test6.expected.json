[
    {
        "file": "Test6.sol",
        "contract": "A",
        "struct": "StructB",
        "slotsSaved": 1,
        "timedout": false
    }
]